test/extended/cli/rsh.go

```diff
@@ -0,0 +1,42 @@
+package cli
+
+import (
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("[cli]oc rsh[Conformance]", func() {
+	defer g.GinkgoRecover()
+
+	var (
+		oc                     = exutil.NewCLI("oc-rsh", exutil.KubeConfigPath())
+		multiContainersFixture = exutil.FixturePath("testdata", "cli", "pod-with-two-containers.yaml")
+		podsLabel              = exutil.ParseLabelsOrDie("name=hello-openshift")
+	)
+
+	g.Describe("", func() {
```

Comment:
Though it has no usage error, empty Describe seems not common in code convention?